samples/stock/demo/pointandfigure/demo.js

```diff
@@ -240,16 +237,22 @@
             takeOrdinalPosition: true,
             finalData: [],
             getColumnMetrics:
-            Highcharts.seriesTypes.column.prototype.getColumnMetrics,
+                Highcharts.seriesTypes.column.prototype.getColumnMetrics,
             markerAttribs,
             generatePnfData,
             translate() {
                 const metrics = this.getColumnMetrics(),
                     calculatedBoxSize = this.calculatedBoxSize;
-                this.markerWidth =
-                metrics.width + metrics.paddedWidth + metrics.offset;
+
                 this.markerHeight =
-                this.yAxis.toPixels(0) - this.yAxis.toPixels(calculatedBoxSize);
+                    this.yAxis.toPixels(0) -
+                    this.yAxis.toPixels(calculatedBoxSize);
+
+                this.markerWidth = this.options.maxPointWidth || Math.min(
```

Comment:
That means setting `maxPointWidth` will force that width. It's not `pointWidth`. `maxPointWidth` should work out of the box, because it's already considered in `getColumnMetrics()`.

Unless you wanted something like `default/undefined = 2x height`? Then it should be more like this:

```ts
width = Math.min(
  pick(maxPointWidth, height * 2),
  metrics.width
  
);
```

Reply:
Yes, you are absolutely right - sorry about that. No comment 🤦🤦🤦